app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss

```diff
@@ -179,6 +179,24 @@
   }
 }
 
+.mortgagecalc__table__lbtt-price {
```

Comment:
Personally I find these class names a little confusing. I have no idea what lbtt and ftb means (doing some digging and I've worked it out). Can you try simplifying the names?

Nice work doing mobile first 👍

Reply:
This is a difficult one, and I'll start by saying I agree with your comment about the names being unclear. In the case of ftb, I'm reluctant to change because throughout this repo in the css, markup, js, translations as well as ruby, `ftb` is used as standard. Regards `lbtt`